README.md

```diff
@@ -87,21 +89,33 @@ This is achieved using AWS CloudFormation. For more details check our [CloudForm
 The controller *will not* manage the security groups required to allow access from the Internet to the load balancers.
 It assumes that their lifecycle is external to the controller itself.
 
-### Discovery
+During startup phase EC2 filter is constructed as follows:
 
-On startup, the controller discovers the AWS resources required for the controller operations:
+* If `CUSTOM_TAG_FILTER` environment variable is set, it is expected to be in the form
+  of `FILTER_NAME=FILTER_VALUE1,FILTER_VALUE2... FILTER2_NAME=FILTER2_VALUE1,FILTER2_VALUE2... ...`.
```

Comment:
The triple dots does not makes sense to me (only the last to show you can add more filters).
I was reading parseFilters() comment and strings.Fields usage, which should be documented here in a user friendly way with an example how to use the custom filer ENV.
I suggest:

If `CUSTOM_TAG_FILTER` environment variable is set, it is expected to be in the form of `CUSTOM_TAG_FILTER="FILTER_NAME=FILTER_VALUE1,FILTER_VALUE2 FILTER2_NAME=FILTER2_VALUE1,FILTER2_VALUE2 ..."`.